app/assets/javascripts/Components/Modals/submission_url_submit_modal.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import Modal from "react-modal";
+
+class SubmitUrlUploadModal extends React.Component {
+  state = {
+    newUrl: "",
+    newUrlAlias: "",
+  };
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    this.props.onSubmit("https://youtu.be/dQw4w9WgXcQ", "");
+  };
+
+  handleUrlChange = event => {
+    this.setState({newUrl: event.target.value});
+  };
+
+  handleUrlAliasChange = event => {
+    this.setState({newUrlAlias: event.target.value});
+  };
+
+  render() {
+    const {isOpen, onRequestClose} = this.props;
+    return (
+      <Modal className="react-modal" isOpen={isOpen} onRequestClose={onRequestClose}>
+        <h2>{I18n.t("submissions.student.create_link")}</h2>
+        <form onSubmit={this.onSubmit}>
+          <div className={"modal-container-vertical"}>
+            <div className={"modal-container"}>
+              <label>
+                {I18n.t("submissions.student.url")}
+                <br />
```

Comment:
In all other modals of this type we put the label to the left of the input. Please do the same here (instead of the label on top of the input)